tests/test_mappings/test_imports/test_modules/test_base.py

```
@@ -10,7 +10,7 @@
 from fyle_integrations_platform_connector import PlatformConnector
 
 from apps.tasks.models import Error
-from apps.mappings.models import ImportLog
+from fyle_integrations_imports.models import ImportLog
 from apps.sage_intacct.utils import SageIntacctConnector
 from apps.mappings.imports.modules.projects import Project
```

Pylint (3.3.7)

[error] Unable to import 'apps.mappings.imports.modules.projects'

(E0401)